lore/features/base.py

```diff
@@ -48,13 +57,22 @@ def version(self, version=str(datetime.date.today())):
         :param version:
         :return:
         """
-        return version
+        return 'v1'
 
+    @property
     def name(self):
-        return inflection.underscore(self.__class__.__name)
+        return inflection.underscore(self._value)
```

Comment:
Not sure I agree here. Given the name will need to be unique for per feature. How would this work, for example, we are writing a query to get conversions, impressions, and conversion_rate?  
If my feature is conversion_rate and want to store conversions and impressions as well.